### PR TITLE
View course

### PR DESCRIPTION
**Pull request by naremanhilles**

### CODE REVIEW

File: Client/src/components/contanier/staff/Courses/AddCourse/index.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  font-size: 34px;
+  font-weight: bold;
+  margin-top: 50px;
+  margin-left: 39px;
+  margin-bottom: 9px;
+}
+.add-course-line {
+  width: 204px;
+  border-bottom: 1.5px solid #06468f;
+  margin-top: 100px;
```
Comment (tomduggan85) [inline Client/src/components/contanier/staff/Courses/AddCourse/index.css:12]: @naremanhilles If you'd like, you could collapse these three margin styles into a single line: `margin: 100px 0 73px 39px`.

File: Client/src/components/contanier/staff/Courses/ViewCourse/index.js [hunk 2]
```diff
@@ -1,11 +1,33 @@
...
+      "Target Level",
+      "Percentage Of Total Numbers"
+    ];
+    const rows = [
+      ["writing and reading", "47/1995", "6", "60%"],
+      ["translating", "4/3/1995", "5", "80%"],
+      ["samerry", "8/9/1995", "3", "70%"],
+      ["sammer", "1/4/1995", "2", "50%"],
+      ["sam", "1/9/1995", "8", "30%"]
+    ];
```
Comment (tomduggan85) [inline Client/src/components/contanier/staff/Courses/ViewCourse/index.js]: @naremanhilles This is a great way to stay productive while the endpoints are being created 👍

File: Client/src/components/contanier/staff/Courses/AddCourse/index.css (hunk 1 above)
Comment (naremanhilles) [reply Client/src/components/contanier/staff/Courses/AddCourse/index.css:12]: fexed it

**Review by kannanhassouna — APPROVED** [review]

**Comment by Fatmasiam** [comment]
@knanahassouna1  
    Can u review this branch I need it to complete another Component that shows details for a specific course

**Review by Fatmasiam — APPROVED** [review]

**Pull request merged**